packages/gatsby-theme-store/src/sdk/orderForm/useMutateAddItems.ts

```diff
@@ -1,18 +1,23 @@
 import { getOrderformId } from './controller/orderForm'
 import type { OrderFormFragment_OrderFormFragment as OrderForm } from './controller/__generated__/OrderFormFragment_orderForm.graphql'
 import type { AddToCartMutationMutationVariables } from './controller/__generated__/AddToCartMutation.graphql'
+import { AddToCartMutation } from './controller/__generated__/AddToCartMutation.graphql'
 
 const getOrderFormController = () => import('./controller')
 
 const mutationAddItem = async ({
   items,
-}: Pick<AddToCartMutationMutationVariables, 'items'>) => {
+  allowedOutdatedData,
+}: {
+  items: AddToCartMutationMutationVariables['items']
+  allowedOutdatedData?: AddToCartMutationMutationVariables['allowedOutdatedData']
+}) => {
```

Comment:
What do you think about this suggestion?

```suggestion
Pick<AddToCartMutationMutationVariables, 'items' | 'allowedOutdatedData' >
```

Reply:
LGTM, will update!

Reply:
wait, how can I make `allowedOutdatedData` optional?

Reply:
even if I force a default value, typescript complains. code:
```ts
const mutationUpdateQuantity = async ({
  items,
  allowedOutdatedData = undefined,
}: Pick<
  UpdateItemsMutationMutationVariables,
  'items' | 'allowedOutdatedData'
>) => {
```

when compiling the following lines
```ts
const items = {}
mutationUpdateQuantity({ items })
```

I get
```
Property 'allowedOutdatedData' is missing in type '{ items: {}; }' but required in type 'Pick<Exact<{ orderFormId: Maybe<string>; items: any; splitItem: Maybe<boolean>; allowedOutdatedData: Maybe<string | string[]>; }>, "items" | "allowedOutdatedData">'.ts(2345)
```

Reply:
It's optional but needs to be explicitly defined, something like that:

```ts
mutationUpdateQuantity({ items, allowedOutdatedData: undefined })
```
It occurs because type Maybe<T> is T | undefined | null

Reply:
you could also use `Partial`, e.g.

```ts
const mutationUpdateQuantity = async ({
  items,
  allowedOutdatedData = [],
}: Partial<Pick<UpdateItemsMutationVariables, 'items' | 'allowedOutdatedData'>>) => {
  // ...
}
```

Reply:
actually, you would need to join two types together, because `items` is supposed to be required, like

```ts
type UpdateQuantityArgs = Pick<UpdateItemsMutationVariables, 'items'> & Partial<Pick<UpdateItemsMutationVariables, 'allowedOutdatedData'>>
```